tests/unit/components/AppHeader.spec.ts

```diff
@@ -1,5 +1,3 @@
-// @vitest-environment nuxt
-
```

Comment:
These comments used to tell vitest to use a nuxt environment for these tests, but after I got stuck for a while trying to debug a test and finally worked out that it wasn't using a nuxt environment, I decided to set that as the default in vitest.config.ts.